panther_analysis_tool/command/migrate.py

```diff
@@ -151,7 +159,7 @@ def migrate(
         disable=analysis_id is not None,
         transient=True,
     ):
-        if spec.analysis_type() == AnalysisTypes.PACK:
+        if analysis_id is None and spec.analysis_type() == AnalysisTypes.PACK:
```

Comment:
If a user provides a single ID, we don't need these extraneous checks. So users can migrate single items at a time.